packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/TransformHelper.java

```diff
@@ -60,6 +60,13 @@ public static void processTransform(ReadableArray transforms, double[] result) {
       return;
```

Comment:
yeah, i think we might need to apply transform-origin here as well. I am not familiar with how this works with fabric layout animations. But we can reuse the multiplying+matrix concat logic here!